keras_cv/layers/preprocessing/grid_mask.py

```diff
@@ -67,26 +67,26 @@ def __init__(
         ratio=0.6,
         rotation_factor=0.1,
         fill_mode="constant",
-        fill_value=1,
+        fill_value=0,
         seed=None,
-        **kwargs
+        **kwargs,
     ):
         super().__init__(**kwargs)
         self._check_input_range(ratio, fill_mode, fill_value)
-        self.random_rotate = layers.RandomRotation(
-            factor=rotation_factor, seed=seed
-        )
+        self.random_rotate = layers.RandomRotation(factor=rotation_factor, seed=seed)
         self.seed = seed
 
     def _check_input_range(self, ratio, fill_mode, fill_value):
-        if ratio not in [ x / pow(0.1, -1) for x in range(0, 10) ]:
-            raise ValueError(f"ratio should be within [0.0, 0.9 ] with 0.1 interval Got {ratio}")
-        
+        if ratio < 0 or ratio > 1:
```

Comment:
@LukeWood if `ratio = 0.999` is given, which isn't valid, then how to check beforehand?

Reply:
Sorry, I may misunderstand.  Why is 0.999 not valid?  can't it be any value in [0, 1]?

Reply:
In the previous implementation, it wasn't working, but I will send an update on your last review. It's fine now.